api/tests/tooling/domain-builder/factory/build-assessment-result.js

```diff
@@ -5,6 +5,7 @@ const Assessment = require('../../../../lib/domain/models/Assessment');
 const buildAssessmentResult = function ({
   id = 123,
   pixScore = 31,
+  reproducibilityRate = 75.5,
```

Comment:
Utile ou pas ? Je ne sais jamais comment fonctionne le séparateur décimal en fonction de la locale client/serveur/GH..
```suggestion
  reproducibilityRate = Number(75.5),
```